backend/src/main/java/hanglog/trip/service/ItemService.java

```diff
@@ -99,17 +99,22 @@ private Place makePlace(final PlaceRequest placeRequest) {
     }
 
     private Place createPlaceByPlaceRequest(final PlaceRequest placeRequest) {
-        // TODO apiCategory를 가지고 category를 탐색
-        final Category category = new Category(1L, "문화", "culture");
-
         return new Place(
                 placeRequest.getName(),
                 placeRequest.getLatitude(),
                 placeRequest.getLongitude(),
-                category
+                findCategoryByApiCategory(placeRequest.getApiCategory())
         );
     }
 
+    public Category findCategoryByApiCategory(final List<String> apiCategory) {
+        final List<Category> categories = categoryRepository.findByEngNameIn(apiCategory);
+        if (categories.size() == 0) {
+            return categoryRepository.findCategoryETC();
+        }
```

Comment:
isEmpty 어때요? 0 매직넘버가 찜찜해서 건의해봅니당

```suggestion
        if (categories.isEmpty()) {
            return categoryRepository.findCategoryETC();
        }
```

Reply:
굿